assets/css/v2/on_bus/time.scss

```diff
@@ -0,0 +1,44 @@
+.departure-time {
+  display: inline-block;
+  align-items: center;
+  justify-content: end;
+  width: 100%;
+  margin-top: 25.5px;
+  font-family: Inter;
+  font-size: 47px;
+  font-style: normal;
+  vertical-align: middle;
+}
+
+.departure-times-with-crowding,
+.departure-time-with-crowding {
+  height: 100%;
+}
+
+.departure-crowding {
+  width: 0;
+  height: 0;
+}
+
+.departure-time__minutes {
+  display: inline-block;
+  width: 88px;
+  height: 100%;
+  font-weight: 800;
+  text-align: right;
+}
+
+.departure-time__minutes-label {
+  display: inline-block;
+  width: 42px;
+  height: 100%;
+  font-weight: 400;
+  text-align: left;
+}
+
+.departure-time__text {
```

Comment:
We have styles here for `minutes` and `text` types, but I think we're missing the `timestamp` type, which would be seen for Commuter Rail departures (see `components/v2/departures/departure_time`).

Reply:
Good catch, added!